### ISSUE TITLE
Names for globals from name section are not used in wasm2wat

### ISSUE THREAD

**Issue by sbc100** [WebAssembly/wabt#1584]
They seem to be ignored:

```
  (global (;15;) i32 (i32.const 172))
  (global (;16;) i32 (i32.const 168))
  (global (;17;) i32 (i32.const 2904))
  (global (;18;) i32 (i32.const 3644))
```

**Comment by 7ombie** [comment]
It looks like WABT only implemented the standardized part of [the Custom Name Section spec][1], which only supports module names, and identifiers for functions and locals (subsections 0, 1 and 2).

The [Extended Name Section Proposal][2] add support for every identifier possible in WAT source (adding subsections 3 through 9).

Can we convert this issue into a feature request?

[1]: https://webassembly.github.io/spec/core/appendix/custom.html#name-section
[2]: https://github.com/WebAssembly/extended-name-section

**Comment by sbc100** [comment]
I think this was added in #1573 and #1659.     So I guess we can close this?

**Comment by sbc100** [comment]
Confirmed this was fixed in #1659.  Sorry I should have closed this already.
